lib/utils/template.js

```diff
@@ -37,20 +37,28 @@ function template(state, options) {
   const error = typeof options.errorHandler === 'function' ?
     options.errorHandler :
     function defaultErrorHandler(errorType) {
-      const path = this.data.toString().replace(/^data/, '');
+      const path = this.data.toString()
+        .replace(/^data/, '');
       const dataPath = path
         .replace(/\['([^']+)'\]/ig, '.$1')
         .replace(/\[(i[0-9]*)\]/ig, '[\'+$1+\']');
       const schemaPath = `#${
         path
           .replace(/\[i([0-9]*)\]/ig, '/items')
           .replace(/\['([^']+)'\]/ig, '/properties/$1')
-      }/${errorType}`;
-
+        }/${errorType}`;
+      function clearDecode(tplString) {
```

Comment:
Not a big thing, but it's better to move out to the outer scope, otherwise it will be created anytime on entering the `defaultErrorHandler()`. I'll do it on master. Thanks for MR, merging